README.md

```diff
@@ -0,0 +1,223 @@
+
+In your Ember app folder, execute:
+
+```
+npx ember-vite-codemod@latest [options]
+```
+
+### options
+
+| Option          | Default | Description                                                                                                                                                                                                                                |
+| :-------------- | :-----: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| --skip-git      |  false  | By default, the process exits if the git repository is not clean. Use this option to execute the command anyway at your own risk.                                                                                                          |
+| --skip-v2-addon |  false  | By default, the process exits if it detects v1 addons you could update or remove before switching to Vite. Use this option to execute the rest of the codemod anyway and discover if Embroider can deal with your v1 addons without issue. |
+| --ts            |  false  | Use this option to indicate your app uses typescript. It will impact what files the codemod creates and the packages it installs.                                                                                                          |
+| --error-trace   |  false  | In case of error, use this option to print the full error trace when it's available.                                                                                                                                                       |
+
+## Steps
+
+The codemod executes a sequence of several ordered tasks. At the end of the process, reinstall your dependencies; if your app follows the standards the codemod expects, it should now builds correctly using Vite. If something went wrong, the sections below detail what the codemod expectations are and what it tries to output so you can figure out how to complete the migration manually.
+
+### Checking for unsupported dependencies
+
+This is a verification task, it doesn't change your code.
+
+The codemod will first check there are no known dependency that is incompatible with the way Vite works. `ember-fetch` needs to go.
+
+### Checking addons are v2
+
+This is a verification task, it doesn't change your code.
+
+The codemod will look at all your Ember dependencies and will advise you for updates.
+
+When you use Embroider in an app that depends on v1 addons, Embroider will try to auto-fix the v1 addons a way that make them compatible with Vite. This approach works for a number of known addons, but we cannot guarantee it will work for any v1 addon you use. The best way to avoid issues is that your classic app already relies only on v2 addons, and the codemod will guide you in that direction:
```

Comment:
```suggestion
When you use Embroider in an app that depends on v1 addons, Embroider will try to auto-fix the v1 addons in a way that makes them compatible with Vite. This approach works for a number of known addons, but we cannot guarantee it will work for any v1 addon you use. The best way to avoid issues is that your classic app already relies only on v2 addons, and the codemod will guide you in that direction:
```